### PR TITLE
Rmse

### PR DESCRIPTION
**Pull request by brookehus**
superposefeaturizer now outputs an array with correct dimensions

### CODE REVIEW

**Comment by cxhernandez** [comment]
You'll have to fix the test as well.

**Comment by mpharrigan** [comment]
Can you describe what the previous behavior was, why it was incorrect, and what the fix does

File: msmbuilder/tests/test_commands.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def test_superpose_featurizer():
                   data_home=get_data_home()))
         ds = dataset('distances')
         assert len(ds) == 10
-        assert ds[0].shape[1] == len(np.loadtxt('all.txt'))
+        assert ds.shape[1] == len(np.loadtxt('all.txt'))
```
Comment (mpharrigan) [inline msmbuilder/tests/test_commands.py]: I'm pretty sure this is wrong. `dataset` objects don't have a shape attribute

File: msmbuilder/tests/test_ghmm.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def test_1():
     indices = topology.select('symbol C or symbol O or symbol N')
     featurizer = SuperposeFeaturizer(indices, trajectories[0][0])
 
-    sequences = featurizer.transform(trajectories)
+    sequences = featurizer.partial_transform(trajectories)
```
Comment (mpharrigan) [inline msmbuilder/tests/test_ghmm.py]: I think transform is (still) appropriate here

Comment (cxhernandez) [inline msmbuilder/tests/test_ghmm.py]: I agree. What is `trajectories` though? A list of `mdtraj.Trajectory` objects?

File: msmbuilder/tests/test_commands.py (hunk 1 above)
Comment (cxhernandez) [inline msmbuilder/tests/test_commands.py]: This is my b. 

@brookehus: In hindsight, this should be:
```
assert ds[0].shape[0] == len(np.loadtxt('all.txt'))
```

**Comment by mpharrigan** [comment]
I'm still confused what this pr is trying to achieve

**Pull request closed without merging**